pennylane/ops/functions/is_commuting.py

```diff
@@ -146,120 +154,69 @@ def check_commutation_two_non_simplified_crot(operation1, operation2):
     control_control = intersection(operation1.control_wires, operation2.control_wires)
     target_target = intersection(target_wires_1, target_wires_2)
 
-    if control_control and target_target:
-        return np.all(
-            np.allclose(
-                np.matmul(operation1.matrix(), operation2.matrix()),
-                np.matmul(operation2.matrix(), operation1.matrix()),
-            )
-        )
-
-    if control_control and not target_target:
+    if control_control:
+        if target_target:
+            return _check_mat_commutation(operation1, operation2)
+        # control_control and not target_target
         return True
 
-    if not control_control and target_target:
-        return np.all(
-            np.allclose(
-                np.matmul(
-                    qml.Rot(*operation1.data, wires=operation1.wires[1]).matrix(),
-                    qml.Rot(*operation2.data, wires=operation2.wires[1]).matrix(),
-                ),
-                np.matmul(
-                    qml.Rot(*operation2.data, wires=operation2.wires[1]).matrix(),
-                    qml.Rot(*operation1.data, wires=operation1.wires[1]).matrix(),
-                ),
-            )
+    if target_target:
+        return _check_mat_commutation(
+            qml.Rot(*operation1.data, wires=operation1.wires[1]),
+            qml.Rot(*operation2.data, wires=operation2.wires[1]),
         )
     return False
 
 
-def check_simplify_identity_commutation(operation1, operation2):
+def check_simplify_identity_commutation(op):
     r"""First check that a parametric operation can be simplified to the identity operator, if it is the case then
      return the commutation relation with the second operation. If simplification is not possible, it returns None.
 
     Args:
-        operation1 (pennylane.Operation): First operation.
-        operation2 (pennylane.Operation): Second operation.
+        op(pennylane.Operation): First operation.
 
     Returns:
          Bool: True if commutation, False non-commmutation and None if not possible to simplify.
     """
```

Comment:
The docstring needs to be adapted to the changes:
- There is no `second operation`.
- `False` is never returned.